articlequality/utilities/extract_labelings.py

```diff
@@ -108,7 +108,8 @@ def extract_labelings(dump, extractor=None, verbose=False):
     :Returns:
         An iterator of dicts containing:
 
-        * page_title -- The normalized title of the article
+        * talk_page_id -- The id of the article's talk page
+        * talk_page_title -- The normalized title of the talk page
```

Comment:
In cases such as the talk page http://pt.wikipedia.org/wiki/?curid=18363, the title in the dump `ptwiki-20200301-pages-meta-history1.xml-p11474p25157.bz2` will be `São Carlos` but since [it was moved](https://pt.wikipedia.org/wiki/Discuss%C3%A3o:S%C3%A3o_Carlos_(S%C3%A3o_Paulo)?diff=57981664) afterwards, this name will be updated to `Discussão:São Carlos (São Paulo)` when we query the API:
https://github.com/wikimedia/articlequality/pull/126/commits/c758aebdb58d783635561f46a6cf217e98686d74#diff-d5d31930d2c85b2a25b02ad7640c4ab6R97

Or do we need to keep track of the old dump title for some reason?

Reply:
It's useful for debugging.  E.g. if we look at an observation, the title can help us know what we're seeing.  Much harder to visually track a number.

Reply:
What should I call it? `dump_talk_page_title`? `talk_page_title_on_dump`? something else?

Reply:
Hmm.  I'd be down for "dump_talk_page_title" or something like that.